config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :photos
```

Comment:
This should probably go after the explanatory comment — otherwise it'll just end up randomly in the middle.